compiler-rt/test/asan/TestCases/Windows/allocator_may_return_null_limits.cpp

```diff
@@ -0,0 +1,34 @@
+// RUN: %clangxx_asan -O0 %s -o %t
+// RUN: %env_asan_opts=allocator_may_return_null=0 not %run %t 2>&1 | FileCheck %s --check-prefix=CHECK-ABORT
+// RUN: %env_asan_opts=allocator_may_return_null=1 %run %t 2>&1 | FileCheck %s --check-prefix=CHECK-RETURN_NULL
+
+// RUN: %clangxx_asan -O0 %s -o %t -DUSER_FUNCTION
+// RUN: %env_asan_opts=allocator_may_return_null=1 %run %t 2>&1 | FileCheck %s --check-prefix=CHECK-RETURN_NULL
```

Comment:
The `%env_asan_opts=allocator_may_return_null=1` part seems redundant here, and defeating the purpose of defining `__asan_default_options`.

Reply:
Whoops, I had meant to set that to `0` and have the user function actually override that `0` to `1`.
But might as well remove it. Done that here: https://github.com/llvm/llvm-project/pull/117929/commits/1f66929b1991827bdd068bd5b8f2c1afe72b129e